src/imports/extras/nodedelegate.cpp

```diff
@@ -167,6 +167,7 @@ void AbstractRectDelegate::updateNode(QSGNode *node, const QModelIndex &index, N
         rectNode->setGradientStops(*stops);
         rectNode->setGradientVertical(nodeGradientOrientation(index, item) == Qt::Vertical);
     } else {
+        rectNode->setGradientStops(*new QGradientStops);
```

Comment:
This would leak memory because an empty QGradientStops is allocated on the heap every time a solid-colored node is updated. Even though it's a small leak, it accumulates quickly for those massive views that have 16000+ nodes...

```
rectNode->setGradientStops(QGradientStops());
```